cypress/integration/community.spec.js

```diff
@@ -0,0 +1,64 @@
+/// <reference types="cypress" />
+// Tests the functionality from the Community member perspective
+
+describe("Community members", () => {
+    beforeEach(() => {
+
+        cy.visit(Cypress.env('login_url'));
+        cy.get("[name=csrfmiddlewaretoken]")
+            .should("exist")
+            .should("have.attr", "value")
+            .as("csrfToken");
+
+        cy.get("@csrfToken").then((token) => {
+            cy.request({
+                method: "POST",
+                url: Cypress.env("login_url"),
+                form: true,
+                body: {
+                    username: "community",
+                    password: "1234567890",
+                },
+                headers: {
+                    "X-CSRFTOKEN": token,
+                },
+            });
+        });
+    })
+
+    it("can mark a translation as good or bad", () => {
+        cy.visit(Cypress.env('home'));
+
+        cy.get('[data-cy="segment-card"]:first')
+            .within(() => {
+
+                cy.get('[data-cy="yes-button"]')
+                    .should('be.visible')
```

Comment:
Cypress “action commands” like `click()` and `type()` (but not `get()`, TIL) already [imply a bunch of assertions](https://docs.cypress.io/guides/core-concepts/interacting-with-elements.html#Actionability) including that the element is visible.